tests/Tests.hs

```diff
@@ -1,6 +1,6 @@
 module Main (main) where
 
-import Criterion.Types
+import Criterion.Types 
```

Comment:
Trailing whitespace.

Reply:
Emacs doesn't show any extra whitespace there.

Reply:
You fixed this in https://github.com/bos/criterion/pull/191/commits/9f5fe87b8746cce56b45b18508911598c03f1e57#diff-0bf78663781e2a6812584ddf0032c386R3, so that part is good now.